backend/src/api/routers/chains.py

```
@@ -60,6 +60,8 @@
         llm = ChatVertexAI(model_name="gemini-1.5-flash", temperature=llm_temp)
     elif os.getenv("GOOGLE_GEMINI") == "1.5_pro":
         llm = ChatVertexAI(model_name="gemini-1.5-pro", temperature=llm_temp)
+    elif os.getenv("GOOGLE_GEMINI") == "2.5_flash":
+        llm = ChatVertexAI(model_name="gemini-2.5-flash", temperature=llm_temp)
```

since 1.0-1.5 models are disabled, could you raise an exception when these models are selected? 
also side note - it would be great if you can add the models 2.0 and above in the if-else.